### PR TITLE
Add a grafana dashboard for boskos resource usage.

### LINKED ISSUE #15303: Outage with Velodrome/boskos logs
**What happened**:
The Velodrome boskos dashboard is returning no data with the following response (in the red ! in the top-left):
```
504 Gateway Time-out

504 Gateway Time-out
nginx/1.10.1
```

**What you expected to happen**:
Data

**How to reproduce it (as minimally and precisely as possible)**:
Currently, visit [the dashboard](http://velodrome.k8s.io/dashboard/db/boskos-dashboard?orgId=1&from=now-24h&to=now). I have confirmed the error is reproducible with different people.

**Please provide links to example occurrences, if any**:
http://velodrome.k8s.io/dashboard/db/boskos-dashboard?orgId=1&from=now-24h&to=now

/kind oncall-hotlist

### PR DESCRIPTION
**Pull request by cjwagner**
__Questions:__ 
1. The `leased` and `other` states were not previously graphed and at a glance they are always 0. Should I remove these from the graphs?
1. I set the default time range to the past day of results. Is that value useful?
1. Do we care what order the graphs are in?
1. Are the colors OK? (I didn't use the automatic colors to ensure that the color schemes are consistent between the graphs.)

Here is what it looks like:
boskos-dashboard

/assign @hongkailiu @stevekuznetsov 
/cc @sebastienvas 
/area boskos

fixes #15303

### CODE REVIEW

File: prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      )
+.addPanels([
+    graphPanel.new(
+        '%s usage status' % resource.friendly,
+        description='The number of %ss in each state.' % resource.friendly,
+        datasource='prometheus',
+        aliasColors={'busy': '#0000ff', 'cleaning': '#00eeff', 'dirty': '#ff8000', 'free': '#00ff00', 'leased': '#ee00ff', 'other': '#ff0000'}
+    )
+    .addTarget(prometheus.target(
+        '%s_busy' % resource.metric,
```
Comment (stevekuznetsov) [inline prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet]: These are the older metrics -- I would suggest

```
sum(boskos_resources{type="<my-named-thing>"}) by (state)
```

Comment (cjwagner) [reply prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet]: We don't have that metric in our data... Sounds like our boskos instance isn't being updated properly? I'll open an issue.

Comment (cjwagner) [reply prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet]: ref: https://github.com/kubernetes/test-infra/issues/11957

Comment (stevekuznetsov) [reply prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet]: Whoa. That was added ages ago. I thought `boskos` was automatically rolled out these days?

**Comment by cjwagner** [comment]
Boskos is now serving the new metric, the grafana dashboards in this PR have been switched to use that. I also made the graphs stack and fill and tweaked the colors to be more readable. Thanks @michelle192837 for the input!
Screenshot from 2019-11-21 14-08-00

**Comment by cjwagner** [comment]
https://monitoring.prow.k8s.io/d/wSrfvNxWz/boskos-resource-usage?orgId=1

**Pull request merged**